scripts/tools/retry.sh

```diff
@@ -9,13 +9,33 @@ function retry {
     local delay="${DELAY:-5}"
     local i=0
 
+    local cr
+    local nl
```

Comment:
I was going to ask for better names… then realized these are actually the full names, I just normally see them in upper case (`CR`/`LF`). Nothing to do here, unless you feel like adding excessive comments (which is kind of pointless).

Reply:
I made them lowercase because they are local variables. I think this is fine.